code/modules/mob/living/death.dm

```diff
@@ -1,22 +1,20 @@
 GLOBAL_VAR_INIT(permadeath, FALSE)
 
-/mob/living/proc/gib(no_brain, no_organs, no_bodyparts)
+/mob/living/proc/gib(no_brain, no_organs, no_bodyparts, unequip_everything)
```

Comment:
This should still unequip by default, also change the microbomb's `gib(1)` to `gib(no_brain = TRUE, unequip_everything = FALSE)`
```suggestion
/mob/living/proc/gib(no_brain, no_organs, no_bodyparts, unequip_everything = TRUE)
```